src/Resources/views/ContactForm/show.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends '@SyliusAdmin/layout.html.twig' %}
+
+{% block title %}{{ 'mango_sylius.contactForm.ui.detailTitle'|trans ({'%Contact%': message.senderName|first })}}{% endblock %}
+
+{% block content %}
+
+
+        <div class="ui segment">
+            {{ 'mango_sylius.contactForm.name'|trans ({'%Name%' : message.senderName})}}
```

Comment:
Always use a lowercase letter in variables

Reply:
DONE